safe_transaction_service/history/tests/utils.py

```diff
@@ -11,7 +11,7 @@ def just_test_if_mainnet_node() -> str:
 
     if not mainnet_node_url:
         pytest.skip("Mainnet node not defined, cannot test oracles", allow_module_level=True)
-    elif requests.get(mainnet_node_url).status_code == 404:
+    elif requests.post(mainnet_node_url).status_code == 404:
```

Comment:
The infura nodes do not respond to get requests. Also, this is how it's done in gnosis_py except for the fact that gnosis-py sends eth_blockNumber request

Reply:
Good catch!